web-portal/components/EventView.vue

```diff
@@ -188,6 +193,26 @@
       venueIsRemote: function () {
         return this.event && this.event.venue && this.event.venue.name === 'Remote Event'
       },
+      venueName: function () {
+        return this.event && this.event.venue && this.event.venue.name
+      },
+      venueAddress: function () {
+        const venue = this.event && this.event.venue
+        let addr = null
+        // abort if street unknown, as that's be the most important part
+        if (venue && venue.street) {
+          addr = venue.street
+          // if city is known, include it too, as well as state
+          // (but don't include state w/o city)
+          if (venue.city) {
+            addr += ' ' + venue.state ? `${venue.city}, ${venue.state}` : venue.city
+            // and only include zip if city, state is known
+            if (venue.zip) addr += ' ' + venue.zip
+          }
+        }
+
+        return addr
+      },
```

Comment:
It looks like we forgot to actually wire this in on the template -- it's still building it up manually (lines 17-19).

Reply:
That was my mistake -- fixed